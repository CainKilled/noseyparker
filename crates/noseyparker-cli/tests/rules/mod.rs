//! Tests for Nosey Parker's `rules` command

use super::*;

/// Check the default list of rules in Nosey Parker using a snapshot test.
/// This will alert us to when the default rules have changed for some reason (usually because a
/// rule has been added).
#[test]
fn rules_list_noargs() {
    assert_cmd_snapshot!(noseyparker_success!("rules", "list"));
}

#[test]
fn rules_list_json() {
    assert_cmd_snapshot!(noseyparker_success!("rules", "list", "--format=json"));
}

#[test]
fn rules_list_jsonl() {
    assert_cmd_snapshot!(noseyparker_success!("rules", "list", "-f", "jsonl"));
}
