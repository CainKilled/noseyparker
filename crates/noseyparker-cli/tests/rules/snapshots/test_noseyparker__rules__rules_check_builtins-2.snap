---
source: crates/noseyparker-cli/tests/rules/mod.rs
expression: stdout
---
106 rules and 2 rulesets: no issues detected

